.github/workflows/tests-and-coverage.yml

```diff
@@ -8,61 +8,27 @@ permissions:
   id-token: write
   contents: read
 
-env:
```

Comment:
are you just moving stuff around in this file? it's hard to tell what change you're trying to make here and/or how it relates to embedding vectors

Reply:
generally, it'd be helpful if you added more context to the PR description describing all of these little changes

Reply:
I remove all the aws related code as we should not be using it

Reply:
As for annotation, the metadata attribute was unused in the api and client